entrypoint/microservice/config/config.py

```diff
@@ -22,6 +22,11 @@ class ProductionConfig(Config):
     # TODO: add required secret configurations
     ENV='production'
     SECRET_KEY=os.getenv('PROD_SECRET_KEY')
+    DB_USER = os.getenv('DEVELOPMENT_DATABASE_USER')
```

Comment:
When the environment variable is set to `production`, the env values should be `'PRODUCTION_*'` not `DEVELOPMENT_*`